src/shared/components/image-button/ImageButton.scss

```diff
@@ -1,61 +1,61 @@
 @import 'src/styles/common';
 
-/* stylelint-disable no-descending-specificity */
-// stylelint doesn't like that the 'imageButton button > svg' selector is more specific than ".default svg" and others;
-// but moving it below the rest of the selectors feels weird
-
-.imageButton {
-  position: relative;
+.wrapper {
   font-size: 0;
+  line-height: 0;
+  position: relative;
+  display: inline-block; // buttons are normally inline-block elements
+  vertical-align: middle;
+}
 
-  svg {
+.imageButton {
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  width: var(--image-button-width, auto);
+  height: var(--image-button-height, auto);
+
+  & > svg {
+    width: var(--image-button-svg-width, 100%);
+    height: var(--image-button-svg-height, 100%);
     transition: opacity 0.4s;
   }
-
-  button {
-    width: 100%;
-    height: 100%;
-  }
-
-  // if a button's immediate child is an svg, it should be the same size as the button
-  button > svg {
-    width: 100%;
-    height: 100%;
-  }
 }
 
 .default {
+  background-color: var(--image-button-default-bg-color, initial);
```

Comment:
Do we need to set this? if we dont specify this will it not take the browser's default

Reply:
Browser is perfectly happy without the fallback. I'm removing it...

![image](https://user-images.githubusercontent.com/6834224/199523635-afae3e73-912d-4402-8045-8e9982b143a9.png)